var/spack/repos/builtin/packages/googletest/package.py

```diff
@@ -36,15 +36,29 @@ class Googletest(Package):
 
     depends_on("cmake", type='build')
 
-    def install(self, spec, prefix):
-        which('cmake')('.', *std_cmake_args)
-
-        make()
+    def cmake_args(self):
+        spec = self.spec
+        if '@1.8.0:' in spec:
+            # New style (contains both Google Mock and Google Test)
+            options = ['-DBUILD_GMOCK=OFF', '-DBUILD_GTEST=ON']
+        else:
+            # Old style (contains only GTest)
+            options = []
+        return options
 
-        # Google Test doesn't have a make install
-        # We have to do our own install here.
-        install_tree('include', prefix.include)
+    def install(self, spec, prefix):
```

Comment:
Optionally, you should also be able to do something like this:
```python
@when('@:1.7.0')
def install(self, spec, prefix):
    install_tree(join_path(self.stage.source_path, 'include'),
        prefix.include)

    mkdirp(prefix.lib)
    install('libgtest.a', prefix.lib)
    install('libgtest_main.a', prefix.lib)
```
That way it uses the default `install` method for newer versions and overrides it only for older versions. But it's up to you whether or not you think this is clearer.